openqa-investigate

```diff
@@ -1,391 +1,178 @@
-#    sources, e.g. infrastructure
-investigate() {
-    local id="${1##*/}"
-    local rc=0
-
-    job_data=$(client-get-job "$id")
-    # shellcheck disable=SC2181
-    [[ $? != 0 ]] && echoerr "unable to query job data for $id: $job_data" && return 1
-    old_name="$(echo "$job_data" | runjq -r '.job.test')" || return $?
-    if [[ "$old_name" =~ ":investigate:" ]]; then
-        post-investigate "$id" "$old_name" || return $?
-        return 0
-    fi
-    clone="$(echo "$job_data" | runjq -r '.job.clone_id')" || return $?
-    if ! "$force" && [[ "$clone" != "null" ]]; then
-        echoerr "Job $id already has a clone, skipping investigation. Use the env variable 'force=true' to trigger investigation jobs"
-        return 0
-    fi
-
-    # determine dependency data or postpone if cluster not done
-    dependency_data=$(query_dependency_data_or_postpone "$id" "$job_data") || rc=$?
-    [[ $rc != 255 ]] && return $rc
-
-    # determine the job in the cluster with the lowest ID to use that for commenting/synchronization
-    first_cluster_job_id=$(echo "$dependency_data" | runjq -r "[$id, [.cluster[] | select(contains([$id]))]] | flatten | min") || return $?
-
-    [[ "$old_name" =~ $exclude_name_regex ]] && echo "Job name '$old_name' matches \$exclude_name_regex '$exclude_name_regex', skipping investigation" && return 0
-    group="$(echo "$job_data" | runjq -r '.job.parent_group + " / " + .job.group')" || return $?
-    [[ "$group" = " / " ]] && [[ "$exclude_no_group" = "true" ]] && echo "Job w/o job group, \$exclude_no_group is set, skipping investigation" && return 0
-    [[ "$group" =~ $exclude_group_regex ]] && echo "Job group '$group' matches \$exclude_group_regex '$exclude_group_regex', skipping investigation" && return 0
-
-    # Optionally we can find "first failed", could extend openQA investigation
-    # method instead for we are just working based on supplied job which can
-    # have more, ambiguous potential changes that we need to bisect on
-
-    # sync by writing initial investigation comment (edited later)
-    rc=0
-    comment_id=$(sync_via_investigation_comment "$id" "$first_cluster_job_id") || rc=$?
-    [[ $rc != 255 ]] && return $rc
-
-    out=$(trigger_jobs "$id" "${@:2}")
-    $verbose && echo "$0, id: '$id', out: '$out'"
-    finalize_investigation_comment "$id" "$first_cluster_job_id" "$comment_id" "$out"
-}
-
-main() {
-    local id=${1:?"Need 'job_id'"}
-    client_prefix=''
-    [ "$dry_run" = "1" ] && client_prefix="echo"
-    set +u
-    if [[ -z "${client_call[*]}" ]]; then
-        client_call=(openqa-cli "${client_args[@]}")
-        client_prefix="${client_prefix:-runcli}"
-        client_call=("$client_prefix" "${client_call[@]}")
-    fi
-    set -u
-    clone_call="${clone_call:-"$client_prefix openqa-clone-job --json-output --skip-chained-deps --max-depth 0 --parental-inheritance --within-instance"}"
-    investigate "$@"
-}
-
-caller 0 > /dev/null || main "$@"
+import requests
```

Comment:
You're missing a shebang here - might help github to detect the used language as well